ur5_moveit_config/launch/moveit_rviz.launch

```diff
@@ -11,6 +12,7 @@
   <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
 	args="$(arg command_args)" output="screen">
     <rosparam command="load" file="$(find ur5_moveit_config)/config/kinematics.yaml"/>
+    <param name="arm_prefix" value="$(arg arm_prefix)"/>
```

Comment:
But with multiple instances of the same plugin it might not work as described/expected. See also discussion in #248.